commit 56843933cb7c270ff8c5b73802044972652992ed
Author: Alexis Ballier <aballier@gentoo.org>
Date:   Fri Jan 18 23:23:32 2013 -0300

    DllSwResample: use uint8_t types for avresample_convert. libav changed their prototypes.

diff --git a/lib/DllSwResample.h b/lib/DllSwResample.h
index 30ee08d..41bf7ad 100644
--- a/lib/DllSwResample.h
+++ b/lib/DllSwResample.h
@@ -113,7 +113,7 @@ public:
   }
   virtual int swr_init(struct SwrContext *s) { return ::avresample_open(s); }
   virtual void swr_free(struct SwrContext **s){ ::avresample_close(*s); *s = NULL; }
-  virtual int swr_convert(struct SwrContext *s, uint8_t **out, int out_count, const uint8_t **in , int in_count){ return ::avresample_convert(s, (void**)out, 0, out_count, (void**)in, 0,in_count); }
+  virtual int swr_convert(struct SwrContext *s, uint8_t **out, int out_count, const uint8_t **in , int in_count){ return ::avresample_convert(s, out, 0, out_count, (uint8_t**)in, 0,in_count); }
 };
 #endif
 
